src/hotspot/share/oops/compressedOops.hpp

```diff
@@ -97,6 +98,12 @@ class CompressedOops : public AllStatic {
   static address  ptrs_base_addr()           { return (address)&_narrow_oop._base; }
   static address  ptrs_base()                { return _narrow_oop._base; }
 
+#if defined(X86) && !defined(ZERO)
```

Comment:
This should not be leaked into shared code

Reply:
Fixed.